.github/workflows/build_tests.yaml

```diff
@@ -49,31 +53,34 @@ jobs:
       run: |
         gcloud config set compute/zone us-east4-a
         gcloud config get compute/zone
+    - name: Install xpk
+      run: |
+        pip install ..
     - name: Create a Pathways-enabled XPK Cluster with 2x v4-8 nodepools. Larger num-nodes to avoid master resizing.
-      run: python xpk.py cluster create-pathways --cluster $TPU_CLUSTER_NAME --tpu-type=v4-8  --num-slices=2 --zone=us-central2-b --default-pool-cpu-machine-type=n1-standard-16 --default-pool-cpu-num-nodes=16 --reservation='${{ secrets.GCP_TPU_V4_RESERVATION }}'
+      run: xpk cluster create-pathways --cluster $TPU_CLUSTER_NAME --tpu-type=v4-8  --num-slices=2 --zone=us-central2-b --default-pool-cpu-machine-type=n1-standard-16 --default-pool-cpu-num-nodes=16 --reservation='${{ secrets.GCP_TPU_V4_RESERVATION }}'
     - name: Authenticate Docker
       run: gcloud auth configure-docker --quiet
     - name: Create test script to execute in workloads
       run: echo -e '#!/bin/bash \n echo "Hello world from a test script!"' > test.sh
     - name: Run a base-docker-image workload
-      run: python xpk.py workload create --cluster $TPU_CLUSTER_NAME --workload $WORKLOAD_NAME  --command "bash test.sh"  --tpu-type=v4-8 --num-slices=2 --zone=us-central2-b
+      run: python main.py workload create --cluster $TPU_CLUSTER_NAME --workload $WORKLOAD_NAME  --command "bash test.sh"  --tpu-type=v4-8 --num-slices=2 --zone=us-central2-b
```

Comment:
I propose we keep the name xpk.py as the entry point, a handful of our documentation and user workflows is build upon xpk.py and would be a large change from calling python3 xpk.py to python3 main.py.

Reply:
I see. At first I also wanted to keep it as `xpk.py` but I encountered some limitation due to python import system. A package cannot have the same name as module within it(in this case `xpk` and `xpk.py`) - see [second answer from this Stackoverflow post](https://stackoverflow.com/questions/54333865/python-no-module-named-error-package-is-not-a-package). At least, this is what happens when running xpk directly using python interpreter. It works correctly when installing first with pip, and then running it as a command. To solve this we have a few alternatives:
### Introduce `main.py` with drawbacks you mentioned
I agree that it's important to be consistent with our documentation 
### Recommend always installing xpk with pip first and discourage running it directly using python interpreter
I like this approach the most. I believe it's a standard practice to use it that way, and it's what other tools do. So from users' perspective, they would need to use it following `README.md`
1. Run `pip install xpk`
2. Run `xpk [subcommand]` 

However, I may not be aware of some cons you are - maybe it would break some customers' workflows?

Another advantage of this approach is that it would be easier to maintain the project, because of simplified e2e tests - we would need to cover only one instead of two ways of installing XPK

### Change name of the package `xpk`
Cons: I believe it's common practice to name packages the same as the project in Python, so we wouldn't be following the convention.


Please let me know what you think

Reply:
We have most users using xpk through python3 xpk.py so changing that flow would require a lot of user awareness :( 

No good answer currently

Reply:
I see, let me investigate a little further then

Reply:
I came up with [the solution](https://github.com/google/xpk/pull/166), PTAL

Reply:
Thanks @PBundyra LGTM